integration-tests/pom.xml

```diff
@@ -125,6 +125,10 @@
                 <!--Manager and key-cloak url-->
                 <event-bridge.manager.url></event-bridge.manager.url>
                 <keycloak.realm.url></keycloak.realm.url>
+
+                <slack.webhook.url>${env.SLACK_WEBHOOK_URL}</slack.webhook.url>
+                <slack.webhook.token>${env.SLACK_WEBHOOK_TOKEN}</slack.webhook.token>
+                <slack.channel>${env.SLACK_CHANNEL}</slack.channel>
```

Comment:
do we really want to enforce the `${env.*}` ?
because that means those need to be set into the local environment even if we don't run any slack scenarios.

or we should document the fact that they need to be set into the `localconfig` file ?

Reply:
Well, if the env is not set then empty value is passed. So it doesn't affect any unrelated test.
About documenting it somewhere, I think that could be done as part of introducing `testlocalconfig`